src/game/tutorial.ts

```diff
@@ -40,7 +40,7 @@ const emptyReserveFleet: ReserveFleet = {
 };
 
 export interface TutorialSetupData {
-  boardState: BoardState;
+  boardState?: BoardState;
```

Comment:
I'd rename this PR something like "tutorials: make `boardState` an optional field", since this isn't just resolving a vs code problem, it's actually a bug in the code that vs code is just letting us know about